core/clients.py

```diff
@@ -751,6 +759,29 @@ async def get_user_info(self) -> Optional[dict]:
                 }
             }
 
+    async def export_backups(self, collection_name: str):
+        coll = self.db[collection_name]
+        documents = []
+        async for document in coll.find():
+            documents.append(document)
+        with open(f"{collection_name}.json", "w") as f:
```

Comment:
Don't write to disk, that may lead to potential issues. Use a `BytesIO` instead.